deployment/terraform-module-knfsd/resources/proxy-startup.sh

```diff
@@ -416,6 +417,17 @@ else
   echo "Metrics are disabled. Skipping..."
 fi
 
+# Enable Knfsd Agent if Configured
+if [ "$ENABLE_KNFSD_AGENT" = "true" ]; then
+
+  echo "Starting Knfsd Agent..."
+  knfsd-agent &
```

Comment:
Might be better to create this as a systemd service so that it can auto-restart and stdout/err will go to the system logs.

The unit file for the service can be copied as part of building the image so that at this point we only need to run:
```
systemctl start knfsd-agent
```

Reply:
Good catch. Have created a systemd service for the agent.